libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -7,27 +7,37 @@ table.calendar-table {
   user-select: none;
   outline: none !important;
```

Comment:
Initial value for `outline` (specifically `outline-style`) is `none` so this declaration should not be necessary.